R/pspViz.R

```diff
@@ -0,0 +1,174 @@
+# figure 4, from the PSP paper, https://journals.asm.org/doi/epub/10.1128/msystems.00847-23
+# code from https://github.com/JRaviLab/psp_app/blob/main/psp_process.Rmd
+
+library(tidyverse)
+library(data.table)
+
+# source("/data/research/jravilab/molevol_scripts/R/cleanup.R")  # in package
+# source("/data/research/jravilab/molevol_scripts/R/summarize.R") # in package
+# source("/data/research/jravilab/molevol_scripts/R/plotting.R") # in package
+# source("/data/research/jravilab/molevol_scripts/R/reverse_operons.R") # in package
+# source("/data/research/jravilab/molevol_scripts/R/networks_domarch.R") # in package
```

Comment:
avoiding hard-coded paths & library calls
```suggestion
```

Reply:
FYI, I didn't commit this change since we'll need the `library()` calls and possibly those commented-out source'd files, but I'll keep it in mind for when we bring the working figure implementation back into the package.